src/github/repository.test.ts

```diff
@@ -101,6 +146,6 @@ describe("createdRepositoryWebhookHandler", () => {
 			}
 		}), DatabaseStateCreator.GITHUB_INSTALLATION_ID, subscription);
 
-		expect(subscription.update).toBeCalledWith({ totalNumberOfRepos: 100 });
+		expect(subscription.update).toBeCalledWith({ totalNumberOfRepos: 2 });
```

Comment:
we check the actual repo count in reposync state so we needed to change this to reflect the mocked reposyncstate count instead of a arbitrary number